build-support/bin/generate_all_lockfiles.sh

```diff
@@ -4,9 +4,20 @@
 
 set -euo pipefail
 
+source $(dirname $0)/../common.sh
```

Comment:
This will not work like you expect when running from `build-support/bin`. Instead, you could either:

1. Expect that you run from `build-support/bin`, so simply do `source build-support/common.sh`. That's fine imo, and we do it elsewhere
2. Use this idiom:

https://github.com/pantsbuild/pants/blob/567ece24e5dade689b00dfd4d9bcb739a8a48f78/build-support/bin/contributors.sh#L3-L7

Reply:
Thanks!